src/WebJobs.Script/Host/IFunctionMetadataManager.cs

```diff
@@ -12,5 +13,7 @@ public interface IFunctionMetadataManager
         ImmutableDictionary<string, ImmutableArray<string>> Errors { get; }
 
         ImmutableArray<FunctionMetadata> Functions { get; }
+
+        IEnumerable<WorkerConfig> LanguageWorkerConfigs { get; set; }
```

Comment:
Can we make this consistent with the other properties and expose an `ImmutableArray` here?